pkg/apis/config/v1beta1/defaults.go

```diff
@@ -95,9 +95,7 @@ func SetDefaultsNodeResourcesAllocatableArgs(obj *NodeResourcesAllocatableArgs)
 
 // SetDefaultsCapacitySchedulingArgs sets the default parameters for CapacityScheduling plugin.
 func SetDefaultsCapacitySchedulingArgs(obj *CapacitySchedulingArgs) {
-	if obj.KubeConfigPath == nil {
-		obj.KubeConfigPath = &defaultKubeConfigPath
-	}
+	// TODO(k/k#96427): get KubeConfigPath and KubeMaster from configuration or command args.
```

Comment:
use the inClusterConfig instead of default KubeConfig Path then we can deploy the scheduler in deployment with serviceaccount.

Reply:
If the user wants to deploy scheduler-plugins as a unified scheduler binary, he can set the kubeconfig path in `KubeSchedulerConfiguration`.  
This issue is that if we set the default KubeConfig Path, we can't set up the scheduler as deployment in https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/manifests/install/charts, because the default path is different with inClusterConfig.